SPECS/python-wheel/python-wheel.spec

```diff
@@ -85,7 +89,8 @@ install -p dist/%{python_wheelname} -t %{buildroot}%{python_wheeldir}
 
 %check
 rm setup.cfg
-PYTHONPATH=%{buildroot}%{python3_sitelib} py.test-3 -v --ignore build
+%{python3} -m pip install pluggy more-itertools
+PYTHONPATH=%{buildroot}%{python3_sitelib} py.test3 -v --ignore build
```

Comment:
Is there a difference between this and `%{python3}`?

Reply:
@MateuszMalisz  `%{python3}` expands to `/usr/bin/python3`. `%{buildroot}%{python3_sitelib}` expands to `/usr/src/mariner/BUILDROOT/python-wheel-0.33.6-12.cm2.x86_64/usr/lib/python3.9/site-packages`.

Let me know if I don't understand your question correctly.

Reply:
Sounds good, I wasn't sure what the PYTHONPATH points to. This is fine.